static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js

```diff
@@ -23,15 +23,41 @@ define([
     };
 
     Fluid250.prototype.create = function () {
-        var templateOptions = {
+
+        var templateOptions, leftMargin, leftPosition, rightPosition, topMargin, videoDesktop;
+
+        templateOptions = {
             showLabel: (this.params.showAdLabel === 'hide') ?
                 'creative__label--hidden' : ''
         };
-        $.create(template(fluid250Tpl, merge(this.params, templateOptions))).appendTo(this.$adSlot);
+
+        leftMargin = (this.params.videoPositionH === 'center' ?
+            -(this.params.videoWidth / 2) : 0
```

Comment:
is it necessary to allow them to configure the size? could just say it has to be a defined size, would safe on setting values in dfp, calculating margins, etc